src/Murder.Editor/Utilities/Gui/ImGuiHelpers.cs

```diff
@@ -453,7 +453,7 @@ public static (bool modified, int result) DrawEnumField(string id, Type enumType
             return (false, 0);
         }
 
-        return (modified, (int)values.GetValue(result)!);
+        return (modified, values.GetValue(result)!);
```

Comment:
interesting, I have fixed this a couple of times so I can see the previous code being a problem. Do you remember how the enum that reproduced this was declared?

Reply:
Sorry, my English is not very good. This example may cause previous code errors:
Public enum MyEnum: byte {}